openbb_platform/platform/provider/openbb_provider/standard_models/calendar_ipo.py

```diff
@@ -0,0 +1,66 @@
+"""IPO Calendar  data model."""
+
+from datetime import (
+    date as dateType,
+    datetime,
+)
+from typing import Optional
+
+from pydantic import Field, field_validator
+
+from openbb_provider.abstract.data import Data
+from openbb_provider.abstract.query_params import QueryParams
+
+
+class CalendarIpoQueryParams(QueryParams):
+    """IPO Calendar Query Params."""
+
+    symbol: Optional[str] = Field(
+        description="Return IPOs with the given ticker (typically the IPO for the company).",
+        default=None,
+    )
+    start_date: Optional[dateType] = Field(
+        description="Return IPOs on or after the given date.",
+        default=None,
+    )
+    end_date: Optional[dateType] = Field(
```

Comment:
So after a discussion with @IgorWounds the start/end _date (and limit?) should use the standardized descriptions here

Reply:
BTW, the query params are not bound by the description in the testing suite.  The conflict is on the data response.

Reply:
@deeleeramone It is checking the standard fields on both the QueryParams and Data. The way it does it is that it doesn't aggregate them which we can improve so you get all the errors in advance instead of 1 by 1. In this example, it will complain about all fields in the QueryParams as none of them used the descriptions.

Reply:
I see.  It would certainly be beneficial to know all errors instead of a one-at-a-time.